docs/guides/cases/connect-mqtt.md

```diff
@@ -42,6 +42,11 @@ data:
         protocolPropertyList:
           MQTTTopic: "/test/test2" 
       ... # You can continue to configure commands and corresponding topics according to your own needs, just continue to add according to this format
+  mutexInstructions: | 
+  # Configure mutexInstructions as required. The format is, mutexInstruction:"description"
```

Comment:
can we illustration why need to config mutexInstructions here?
If this config is optional, need to emphasis the optional choice than the required words.